src/app/organisms/room/RoomTimeline.tsx

```diff
@@ -989,6 +991,70 @@ export function RoomTimeline({ room, eventId, roomInputRef, editor }: RoomTimeli
     [editor]
   );
 
+  // States used for swipe-left-reply. Used for animations and determining whether we should reply or not.
+  const [isTouchingSide, setTouchingSide] = useState(false);
```

Comment:
Could we refactor this touch event handling into a custom hook?

Reply:
Separated to `useSwipeLeft` https://github.com/cinnyapp/cinny/pull/1647/commits/677ad454ef2f434b6cd08958be1e6930182657a3